compiler/modules/front/const.bal

```diff
@@ -442,6 +442,14 @@ function foldTypeCastExpr(FoldContext cx, t:SemType? expectedType, s:TypeCastExp
                 value = <float>value;
             }
         }
+        else if toNumType == t:DECIMAL {
+            if value is int {
```

Comment:
`value is int|float`

Change the ones above to use unions rather than `||`.

Reply:
Changed